gr-azure-software-radio/examples/README.md

```diff
@@ -70,4 +71,44 @@ To run the flowgraph, you must:
 
 Run the flowgraph and you should see the QT GUI Sink block showing the contents of your blob object.
 
-Once you are done with running the examples, delete your blob object to stop being charged for storage. 
+Once you are done with running the examples, delete your blob object to stop being charged for storage.
+
+# Event Hub Examples
+## Event Hub Example Prerequisites
+To run [eventhub_sink_example.grc](../examples/eventhub_sink_example.grc) or [eventhub_source_example.grc](../examples/eventhub-source-example.grc), you must first do the following:
+1. Create an Event Hub in your Azure subscription
+    - See: [Create an Event Hub](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create) for instructions.
+2. Choose how to authenticate to the Azure Event Hub. This example uses the "connection string" authentication option.
+    - See [Get an Event Hub Connection String](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string) for instructions on how to obtain a connection string.
+
+## Event Hub Sink Example
+If you plan to use the Azure CLI to authenticate to the back end, please run
+
+```bash
+az login
+```
+
+and then launch GNU Radio Companion from the same terminal. This will ensure your authentication tokens are available when running the flowgraph. Open [eventhub_sink_example.grc](../examples/eventhub_sink_example.grc).
+
+To run the flowgraph, you must:
+- change the connection_str variable to use your connection string
+- change the eventhub_name variable to use the event hub entity you created as part of the [Event Hub Prerequisites section above](#event-hub-example-prerequisites)
```

Comment:
I couldn't get the eventhub source example to start returning messages until I went into the portal and created the consumer group ID I wanted to use. I think we need to add a step here to do that.